include/libtorrent/storage.hpp

```diff
@@ -286,7 +286,7 @@ namespace libtorrent
 		, boost::noncopyable
 	{
 		// hidden
-		storage_interface(): m_settings(0) {}
+		storage_interface() {}
```

Comment:
I think it would make sense to remove this. it's unnecessary now. (please remove the // hidden comment too)